x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_list/credentials_list.tsx

```diff
@@ -113,6 +119,24 @@ export const CredentialsList: React.FC = () => {
     hidePerPageOptions: true,
   };
 
+  if (items.length < 1) {
```

Comment:
Just curious - why put this up here above the `onTableChange` fn? It feels a bit odd location-wise - I'd either put it as an early return or I'd put it together with the final return.

I'd personally lean towards just combining this w/ the final return as a ternary on line 145:

```js
return items.length ? (
  <EuiBasicTable />
) : (
  <EuiEmptyPrompt />
);
```
LMK if that makes sense to you!

Reply:
https://github.com/elastic/kibana/pull/80034/commits/a6be4b2aaaac3265ddb8792fde8c4f70ba35059f